platforms/cpuinfo.go

```diff
@@ -129,3 +136,18 @@ func getCPUVariant() string {
 
 	return variant
 }
+
+// getCPUArchitecture from uname -m
+// when getCPUInfo func from /proc/cpuinfo read error,
+// then call getCPUArchitecture func get Cpu Architecture
+func getCPUArchitecture() (string, error) {
+	if !isLinuxOS(runtime.GOOS) {
```

Comment:
Is it Linux specific? https://pubs.opengroup.org/onlinepubs/009604599/functions/uname.html is in POSIX.

Reply:
No,macOS have this spec;current func is only by linux use, we can remove this check.